cmd/account/cli.go

```diff
@@ -159,11 +159,12 @@ func (o *cliOptions) run() error {
 	}
 
 	if o.output == "env" {
-		fmt.Printf("AWS_ACCESS_KEY_ID=%s \nAWS_SECRET_ACCESS_KEY=%s \nAWS_SESSION_TOKEN=%s \nAWS_DEFAULT_REGION=%s\n",
+		fmt.Printf("AWS_ACCESS_KEY_ID=%s\nAWS_SECRET_ACCESS_KEY=%s\nAWS_SESSION_TOKEN=%s\nAWS_DEFAULT_REGION=%s\nAWS_REGION=%s\n",
```

Comment:
I see [line 52](https://github.com/openshift/osdctl/pull/313/files#diff-c20c31ee54301dc0f1a15f46ffc55b79d7dc3e86db5723388b5290caf1163658R152) using `%q`. While I don't think these values will ever have a space in them, should I make that change in this PR as well?